neptune/client_config.py

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
This is an internal implementation detail. It should be in `neptune/internal/backends`